Modules/TOF/src/TrendingRate.cxx

```diff
@@ -97,8 +97,13 @@ void TrendingRate::computeTOFRates(TH2F* h, std::vector<int>& bcInt, std::vector
     return;
   }
```

Comment:
This is the point where the current version crashes.
If nb==0 -> hback is not created and then you cannot ask for integral (crash)

Please add this protection here
  if (nb == 0) { // threshold too high? return since hback was not created
    delete hpdiff;
    return;
  }